3_Automation/06_Bestiary.yml

```diff
@@ -43,45 +43,30 @@ Name:                                 # must be unique
   Type: Companion, level 1
   HP: 1
   Passives:
-    Keen Eyed: When their Dealer makes a Detection check as an action with the companion present, they do so with Upper Hand
+    Keen Eyed: 
```

Comment:
Semicolons require values. The semicolon declares 'this is a key, look at the next item for the value'. In this case, it seems like `Keen Eyed` is the value that we then need to look @ the other database for. With the decision to centralize all items, should Bestiary then make the distinction between Passive and Action? I'm thinking not. It should just be a Powers heading under the creature

Reply:
Bestiary won't make a distinction between Passive and Action in a future version, removed instances of dangling semicolon in the doc.